src/dev-app/tree/tree-demo.html

```diff
@@ -1,22 +1,11 @@
 <mat-accordion class="demo-tree-container">
-
   <mat-expansion-panel>
-    <mat-expansion-panel-header>Flattened tree</mat-expansion-panel-header>
-    <mat-tree [dataSource]="dataSource" [treeControl]="treeControl">
-      <mat-tree-node *matTreeNodeDef="let node" matTreeNodeToggle matTreeNodePadding>
-        <button mat-icon-button disabled></button>
-        {{node.filename}} : {{node.type}}
-      </mat-tree-node>
-      <mat-tree-node *matTreeNodeDef="let node;when: hasChild" matTreeNodePadding>
-        <button mat-icon-button matTreeNodeToggle
-                [attr.aria-label]="'toggle ' + node.filename">
-          <mat-icon class="mat-icon-rtl-mirror">
-            {{treeControl.isExpanded(node) ? 'expand_more' : 'chevron_right'}}
-          </mat-icon>
-        </button>
-        {{node.filename}} : {{node.type}}
-      </mat-tree-node>
-    </mat-tree>
+    <mat-expansion-panel-header>Flat tree</mat-expansion-panel-header>
```

Comment:
Rather than listing out the examples individually, you can use the `material-example-list` component to render out all of them. This has the advantage that it will include any new example that we add. You can see how it's being used in the `table-demo.ts`.